packages/ckeditor5-table/src/tablecolumnresize/tablecolumnresizeediting.js

```diff
@@ -589,23 +475,25 @@ export default class TableColumnResizeEditing extends Plugin {
 			return;
 		}
 
+		this._moved = true;
```

Comment:
What is this property? I don't see it used anywhere - looks like a leftover to me.